modules/ldap-syncing-rfc2307.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-rfc2307_{context}"]
+= Syncing groups using the RFC 2307 schema
+
+For the RFC 2307 schema, the following examples synchronize a group named `admins` that has two
```

Comment:
This intro is a repeat of what was already said in "LDAP group sync examples". Does it need to be said again?

Reply:
Outdated